snapcraft.yaml

```diff
@@ -30,29 +43,39 @@ apps:
       - network-bind
 #      - gnome-3-26-1604
 parts:
-#  desktop-gnome-platform:
-#    stage:
-#      # We override this in hexchat-copy with gtk2 version
-#      - -flavor-select
-  meson:
-    source: https://github.com/mesonbuild/meson/releases/download/0.45.0/meson-0.45.0.tar.gz
-    plugin: python
-    prime: [-*]
+  desktop-gtk2:
+    source: https://github.com/ubuntu/snapcraft-desktop-helpers.git
+    source-subdir: gtk
+    plugin: make
+    make-parameters: ["FLAVOR=gtk2"]
+    build-packages:
+      - libgtk2.0-dev
+    stage-packages:
+      - libxkbcommon0  # XKB_CONFIG_ROOT
+      - shared-mime-info
+      - libgtk2.0-0
+      - libgdk-pixbuf2.0-0
+      - libglib2.0-bin
+      - libgtk2.0-bin
+      - unity-gtk2-module
+      - locales-all
+      - libappindicator1
```

Comment:
```suggestion
```

Seems like a random package.

Reply:
Those are for unity7, but it could work without those.

Reply:
I dropped libappindicator and unity-gtk2-module as well as cleaned up some other things